addons/account/static/src/components/systray_item/systray_item.js

```diff
@@ -0,0 +1,26 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+import { session } from '@web/session';
+
+export class AccountSystrayItem extends Component {
+    setup() {
+        this.currentCompany = useService("company").currentCompany;
+        this.is_quick_edit_mode_enabled = session.is_quick_edit_mode_enabled;
```

Comment:
While we use the snake_case convention for Python, we use camelCase in JS
```suggestion
        this.isQuickEditModeEnabled = session.is_quick_edit_mode_enabled;
```